### ISSUE TITLE
How to get custom banner to server and custom sounds? Also, how to make custom client?

### ISSUE THREAD

**Issue by gtdk** [GrowtopiaNoobs/GrowtopiaServer#1198]
Hi! 

Can someone help me with custom banner, custom sounds and custom client?

Thanks

**Comment by DrOreo002** [comment]
Thats too much to ask actually 😂😂

But anyway, this might be the correct answer so correct me if I'm wrong
1. Custom banner is just a replacement of .rttx file, not sure where its located it might be somewhere on the cache folder or items.dat
2. Custom sounds?, well I think you cann
 just replace the already existing sounds 🤔
3. Custom clients?, you mean new Growtopia?, oh boy you gotta learn how to make a game for that. Anyway the cpp bot sounds promising also, maybe you can make simple GUI working out of the packets that gt send?, like when it joins the server will gave you world listing and news, just a little bit of brain cells should help you make a simple gui out of that information

**Comment by DEERUX** [comment]
- using custom cache and custom items.dat which our gtv3 team should have released
- learn how to make a game first if you are talking about sone custom client like we have.

**Comment by gtdk** [comment]
@DEERUX Hi! Would u send me gtv3 custom things to discord? Restark#0001. Thanks

**Comment by DEERUX** [comment]
they are published already at iProgramInCpp's github or if you want the json version look at dark's github and use it with anybodys item decoder.

**Comment by gtdk** [comment]
And with custom thing i meant Custom exe like U can connect with it to ur server

**Comment by DEERUX** [comment]
nah I am not going to share the custom client to anyone just gave you a tip how to make custom items in private servers

**Comment by gtdk** [comment]
aight
i found them
https://github.com/iProgramMC/Growtopia_ItemsDatReader
https://github.com/iProgramMC/Growtopia_ItemsDatEncoder

**Comment by gtdk** [comment]
wat i can do w them'

**Comment by DEERUX** [comment]
yes exactly, they were published after gtv3 0.15 source leak so we made it possible first for anyone to get custom items and you can basically decode and encode items.dat, from .dat to .txt and vice versa

**Comment by gtdk** [comment]
but what are you doing w gtv3 at moment

**Comment by DEERUX** [comment]
its a private server which clones real gt now and nothing else I have showcased proper consumables and /renderworld in my youtube channel: DEERUX

**Comment by gtdk** [comment]
what the hec
how is posible

**Comment by DEERUX** [comment]
its just a lot of hardwork and learning/education thats how its possible

**Comment by gtdk** [comment]
aight i downloading files, then i make like enet server test?

**Comment by DEERUX** [comment]
idk, published gtv3 rn is 0.15 and kinda old, for the items.dat reader/encoder u dont need any „enet server“ at all it works right out of the box made in c#

**Comment by gtdk** [comment]
and how can i make custom banner works

**Comment by DEERUX** [comment]
you can use a png to rttex converter make sure the resolution is perfect and then you can just use custom caching and make it so gt downloads the newest banner from „interface“ folder automatically

**Comment by gtdk** [comment]
where should i put custom cache

**Comment by gtdk** [comment]
i put them to gtps files folder

**Comment by gtdk** [comment]
i made folder
CDNContent49
then i put cache there

**Comment by DEERUX** [comment]
find it out yourself if you wanna learn something, you only have to tell gt directory over the onsupermain packet instead of ubisoft's website and link it to yours

**Comment by DEERUX** [comment]
yeah thats correct at least I guess find the rest out yourself I am busy

**Comment by gtdk** [comment]
aight
deerux

**Comment by gtdk** [comment]
i got that items dat encoder etc

**Comment by DrOreo002** [comment]
Is this GuckTubeYT number 2? xDDDDDD

anyway,

this is an issue tab, place to post an issue related to the repo. Not a place to ask for a "spoon feed", also good luck on ur research

**Comment by gucktubeyt2** [comment]
> Is this GuckTubeYT number 2? xDDDDDD
> 
> anyway,
> 
> this is an issue tab, place to post an issue related to the repo. Not a place to ask for a "spoon feed", also good luck on ur research

lol, why me
